clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/handlers/MigrateServerGroupStrategy.java

```diff
@@ -159,6 +159,7 @@ public synchronized MigrateServerGroupResult generateResults(ServerGroupLocation
       deployDescription.setKeyPair(keyPair != null ? keyPair : launchConfig.getKeyName());
       deployDescription.setAssociatePublicIpAddress(launchConfig.getAssociatePublicIpAddress());
       deployDescription.setCooldown(sourceGroup.getDefaultCooldown());
+      deployDescription.setGroupMetrics(sourceGroup.getGroupMetrics());
```

Comment:
@anotherchrisberry Eek, it is safe to omit. Thought I squashed it away. Group metrics are a separate API call and not on the model at all. AWS doesn't expose group metrics on the ASG at all.

Need to fix the copy spec failure too.